pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -396,6 +396,11 @@ func AddAlertImageMigrations(mg *migrator.Migrator) {
 			{Cols: []string{"token"}, Type: migrator.UniqueIndex},
 		},
 	}
+
 	mg.AddMigration("create alert_image table", migrator.NewAddTableMigration(imageTable))
 	mg.AddMigration("add unique index on token to alert_image table", migrator.NewAddIndexMigration(imageTable, imageTable.Indices[0]))
+
+	mg.AddMigration("alter alert_image.url to mediumtext in mysql and text in postgres", migrator.NewRawSQLMigration("").
```

Comment:
```suggestion
	mg.AddMigration("support longer URLs in alert_image table", migrator.NewRawSQLMigration("").
		Default("ALTER TABLE alert_image ALTER COLUMN url TYPE VARCHAR(2048);").
		Mysql("ALTER TABLE alert_image MODIFY url VARCHAR(2048) NOT NULL;"))
```

Reply:
Thanks, good catch the nullable would have been dropped for MySQL :)

Reply:
I didn't use the `Default(...)` as suggested since SQLite does not support `ALTER COLUMN`, we would need to introduce more migration steps and since the column in SQLite already support long text it seems unnecessary. Let me know what you think.